### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by endrju70** [rustdesk/rustdesk#5605] [Q&A]
machine translation used: hello, I'm a pro version user. Unfortunately, I was too reckless when I logged in to computers as admin and now I don't have access to these computers. I keep getting a message in the console that my license is used up and I can't do anything. I am asking for help and advice on how to reset the settings to regain access in the console. Thank you

**Comment by rustdesk** [comment]
mail to support@rustdesk.com please. @dinger1986

**Reply by dinger1986** [reply]
Can help via email if it needs more than just the license revoked 👍

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#invoices-and-migration
